test/hello.e2e.spec.ts

```diff
@@ -1,31 +1,30 @@
-import * as request from 'supertest';
```

Comment:
Can you revert any code style changes and leave only actual code changes.

Reply:
Ah, sorry, editor did it for me 😅